src/ducks/settings/HarvestBankAccountSettings.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { Q, Query } from 'cozy-client'
+
+import Modal from 'cozy-ui/transpiled/react/Modal'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+
+import AccountModal from 'cozy-harvest-lib/dist/components/AccountModal'
+import { MountPointContext } from 'cozy-harvest-lib/dist/components/MountPointContext'
+
+const makeQuerySpecForAccountTriggers = accountId => ({
+  query: () =>
+    Q('io.cozy.triggers').where({
+      'message.account': accountId
+    }),
+  as: `account/${accountId}/triggers`
+})
+
+const HarvestSpinner = () => {
+  return (
+    <div className="u-m-2 u-ta-center">
+      <Spinner size="xxlarge" />
+    </div>
+  )
+}
+
+const HarvestLoader = ({ accountId, children }) => {
+  return (
+    <Query query={Q('io.cozy.accounts').getById(accountId)}>
+      {({ data: account, fetchStatus }) => {
+        if (fetchStatus === 'loading') {
+          return <HarvestSpinner />
+        } else {
+          const konnectorSlug = account.account_type
+          return (
+            <Query query={Q('io.cozy.konnectors').getById(konnectorSlug)}>
+              {({ data: { attributes: konnector }, fetchStatus }) => {
+                if (fetchStatus === 'loading') {
```

Comment:
shouldn't we check if there is no error too?

Reply:
Done